library/src/validations/email/email.ts

```diff
@@ -2,15 +2,16 @@ import type { ErrorMessage, PipeResult } from '../../types.ts';
 import { getOutput, getPipeIssues } from '../../utils/index.ts';
 
 /**
- * Creates a validation functions that validates a email.
+ * Creates a validation function that validates a email.
  *
  * @param error The error message.
  *
  * @returns A validation function.
  */
 export function email<TInput extends string>(error?: ErrorMessage) {
   return (input: TInput): PipeResult<TInput> =>
-    !/^[\w+-]+(?:\.[\w+-]+)*@[A-Z0-9]+(?:(?:\.|-)[A-Z0-9]+)*\.[A-Z]{2,}$/i.test(
+    // eslint-disable-next-line security/detect-unsafe-regex -- false positive according to https://devina.io/redos-checker
```

Comment:
Do we know why the regex is detected as unsafe? Can we check it again with another tool to be sure?

Reply:
See also the comment below, safe-regex is just buggy (I've triple checked with other JS ReDoS detectors)

Reply:
removed